packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -1282,4 +1286,31 @@ const retry = async <T>(
  * @returns void
  */
 const sleep = async (milliseconds: number): Promise<void> => new Promise((resolve) => setTimeout(resolve, milliseconds));
+
+/**
+ * Imports an existing DDB table.
+ * @param tableDef Tabel definition of imported table.
+ */
+const importTable = async (tableDef: CustomDDB.Input): Promise<AWSCDKAsyncCustomResource.OnEventResponse> => {
+  // TODO: Add import validation
+  console.log('Initiating table import process');
+  console.log('Fetching current table state');
+  console.log(`Table name: ${tableDef.tableName}`);
```

Comment:
minor nit: (since you already have a TODO up there):
Can combine above two lines to `console.log('Fetching current state of table ${tableDef.tableName}');` to make it more readable in the logs.

Reply:
I'll update this in the next PR.